css/my_portfolio.css

```diff
@@ -99,3 +99,249 @@ header {
   top: 23em;
   left: 60%;
 }
+
+#work-section {
+  text-align: center;
+}
+
+.card-work {
+  margin-top: 26%;
+}
+
+.card-header-name {
+  font-size: 2rem;
+  color: #1c1a19;
+}
+
+.card-header-border {
+  width: 80%;
+  border: 1px solid #212121;
+  margin: 9%;
+}
+
+.cards {
+  margin-top: 17%;
+}
+
+.card-name {
+  margin-right: 27%;
+}
+
+.card-text {
+  text-align: left;
+  margin-left: 4%;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  width: 95%;
+}
+
+.skills {
+  width: 95%;
+}
+
+.skill1 {
+  margin: 4% 1% 4% 4%;
+  border: 1px solid #8993a4;
+  width: 4em;
+  background: #fff;
+}
+
+.skill2 {
+  margin: 4% 1%;
+  border: 1px solid #8993a4;
+  width: 4em;
+  background: #fff;
+}
+
+.skill3 {
+  margin: 4% 1% 4%;
+  border: 1px solid #8993a4;
+  width: 8em;
+  background: #fff;
+}
+
+.skill4 {
+  margin: 4% 1% 4% 1%;
+  border: 1px solid #8993a4;
+  width: 4em;
+  background: #fff;
+}
+
+.card-button {
+  float: left;
+  width: 29%;
+  margin: 2% 4%;
+  height: 3em;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 24px;
+  background: #ff6b00;
+  border: #ff6b00;
+  text-align: center;
+  color: #fff;
+}
+
+button:hover {
+  background: #e05304;
+  border: 1px solid #f55800;
+  color: #f4f1f0;
+}
+
+.btn-cards {
+  width: 100%;
+  float: left;
+  margin-left: 0;
+  height: 4.1em;
+  color: #fff;
+}
+
+.card-background-image {
+  background-image: url("../assets/images/Img_Placeholder2.png");
+  height: 24em;
+  width: 92%;
+  margin: 8% 4%;
+}
+
+.card-headline {
```

Comment:
- [ ] Hi there, you have really done so well, but I think you still have not applied the right fonts. Please click on the text in the Figma design and you will see the applied properties to the left. Check screenshot below of your design on the left and [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) on the right. 
- [ ] Kindly take a closer look at your design and the Figma design and fix the necessary fonts
![image](https://user-images.githubusercontent.com/62849872/143580876-cd8920c5-9e38-4952-b0e1-ae54aaec175b.png)

Reply:
The font: "Crete Round" was rejected by linter; That's why I used font-family: "Crete Round", sans-serif"